runtime/android/core/src/org/xwalk/core/XWalkInitializer.java

```diff
@@ -11,13 +11,62 @@
 import org.xwalk.core.XWalkLibraryInterface.InitializationListener;
 
 /**
- * <code>XWalkInitializer</code> is a helper to initialize the Crosswalk environment as
+ * <code>XWalkInitializer</code> is a helper to initialize the Crosswalk Project runtime as
  * {@link XWalkActivity}. But unlike {@link XWalkActivity}, <code>XWalkInitializer</code>
- * neither provide UI interactions to help the end-user to download the appropriate
- * Crosswalk library, nor prompt messages to notify the specific error during initializing.
- * Due to this limitation, <code>XWalkInitializer</code> only works for embedded mode, or
- * for shared mode if the appropriate Crosswalk library has already been installed on the
- * device.
+ * neither provide UI interactions to help the end-user to download appropriate Crosswalk Project
+ * runtime, nor prompt messages to notify specific errors during initializing. Due to this
+ * limitation, <code>XWalkInitializer</code> only works for embedded mode, or for shared mode
+ * if appropriate Crosswalk Project runtime has already been installed on the device.
```

Comment:
How about mention depressing because lzma is added in master as well.

`download appropriate Crosswalk Project runtime` to `download or extract or extract appropriate Crosswalk Project runtime`